Alkousa/requirements.txt

```diff
@@ -0,0 +1,9 @@
+libraries:
+
```

Comment:
Remove this, see https://pip.pypa.io/en/stable/user_guide/#requirements-files

Reply:
done